README.md

```diff
@@ -1 +1,19 @@
 # mysql-router-container
+
+OCI image for MySQL Router operator.
+
+Built for usage in the [MySQL Router Charmed Operator for k8s](https://github.com/canonical/mysql-router-container).
+
+## Build
+
+```shell
+docker build -t mysql-router:latest .
+```
+
+## Run
```

Comment:
do we need the `Run` and `Environment` sections if they dont contain any content?

Reply:
Same comment as in [here](https://github.com/canonical/mysql-router-container/pull/1#discussion_r838941594). Depends what will be decided for `run.sh`